src/steCRV.t.sol

```diff
@@ -38,9 +38,9 @@ contract MockOracle {
 contract ETHstETHPoolTest is DSTest {
 
     uint256 constant WAD = 10**18;
-    address constant REGISTRY   = 0x7D86446dDb609eD0F5f8684AcF30380a356b2B4c;
-    address constant POOL       = 0xDC24316b9AE028F1497c275EB9192a3Ea0f67022;
-    address constant ETH_ORACLE = 0x64DE91F5A373Cd4c28de3600cB34C7C6cE410C85;
+    address constant ADDRESS_PROVIDER = 0x0000000022D53366457F9d5E68Ec105046FC4383;
```

Comment:
Checking `0x0000000022D53366457F9d5E68Ec105046FC4383.get_registry()` it returns another contract address than `0x7D86446dDb609eD0F5f8684AcF30380a356b2B4c`. Is it because actually the registry changed during this time?

Reply:
Yes, @kmbarry1 noticed that.